pkgs/test/stdenv/no-broken-symlinks.nix

```diff
@@ -22,6 +22,10 @@ let
     ln -s${if absolute then "r" else ""} "$out/valid" "$out/valid-symlink"
   '';
 
+  mkValidSymlinkOutsideNixStore = absolute: ''
+    ln -s${if absolute then "r" else ""} "/etc/my_file" "$out/valid-symlink"
```

Comment:
The condition is inverted here, right?

Reply:
I don't believe so, at least from what I understand from the man-page (https://man7.org/linux/man-pages/man1/ln.1.html)

> `ln [OPTION]... [-T] TARGET LINK_NAME`
> ...
> In the 1st form, create a link to `TARGET` with the name `LINK_NAME`.

```
$ ln -s /etc/coolstuff mylink
$ ls -la mylink 
lrwxrwxrwx 1 cbaker cbaker 13 Jan 23 18:03 mylink -> /etc/coolstuff
```

Reply:
Right, so it’s absolute by default and `-r` would make it relative? So `if absolute then "" else "r"` would be correct? Am I misunderstanding?